### ISSUE TITLE
2 roboclaws?

### ISSUE THREAD

**Issue by alduxvm** [sonyccd/roboclaw_ros#25]
Hi!

Thanks for this great work! This is not an issue with this repo... which works great to be honest! 

I have a couple of questions if its possible to use this node to control 2 roboclaw boards... my robot has 4 wheels, 4 motors, 4 encoders and 2 roboclaw boards.

For a robot as follows:

[ a ]-------[ b ]
|..................... |
| .....................|
[ c ]-------[ d ]

I currently have one roboclaw controlling motor a and c, the other one controls b and d.

Of course if I use directly this node I can only control one side :P 

what do you think?

Thanks a lot!!

**Comment by DariusStu** [comment]
You may want to check out this other version of a roboclaw node -->https://github.com/sheaffej/roboclaw_driver. He uses a 4-wheel differential drive base that drives two Roboclaw controllers.

**Comment by alduxvm** [comment]
oh yes, I tried that already but it does not work and the instructions never mention how to configure two boards... I already contacted the author to see if he can help, but I did that like 2 months ago... I'm happy that you replied fast!

**Comment by jmorrow10** [comment]
> 

@alduxvm did you ever figure out how to set up 2 boards?  The author still hasn't gotten back to me either

**Comment by alduxvm** [comment]
@jmorrow10 yes, I made it work, I created two workspaces (basically two copies) one for wheels A-B and another for C-D, with changes to the code to adapt and avoid conflicts in topics and thats it! it worked very good! I was very happy with my robot, but I needed more power, so I move to larger motors and the odrive motor controllers...
